### PR TITLE
Update AWS CSI driver

### PR DESCRIPTION
**Pull request by kron4eg**
**What this PR does / why we need it**:
* Updated manifests
* Updated image versions
* Added CSI Snapshot webhook

**Which issue(s) this PR fixes**:

xRef #2835

**What type of PR is this?**
/kind chore

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
* Update AWS CSI driver
* Added AWS CSI Snapshot webhook
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: addons/csi-aws-ebs/csi-snapshot-webhook.yaml [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  clientConfig:
+    service:
+      namespace: kube-system
+      name: "snapshot-validation-service"
+      path: "/volumesnapshot"
+    caBundle: |
+{{ .Certificates.KubernetesCA | b64enc | indent 6 }}
+  admissionReviewVersions: ["v1"]
+  sideEffects: None
+  failurePolicy: Ignore # We recommend switching to Fail only after successful installation of the webhook server and webhook.
```
Comment (kron4eg) [inline addons/csi-aws-ebs/csi-snapshot-webhook.yaml:69]: We need to discuss this.

Comment (kron4eg) [reply addons/csi-aws-ebs/csi-snapshot-webhook.yaml:69]: @xmudrii this is changed from previously "Fail", question is, should we change it back to the Fail or leave it along?

Comment (xmudrii) [reply addons/csi-aws-ebs/csi-snapshot-webhook.yaml:69]: @kron4eg I don't have strong opinions, maybe reverting it to Fail is a safer approach

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Pull request merged**